src/GraphQL/Execution/ParsedError.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GraphQL.Execution
+{
+    /// <summary>
+    /// Represents the fields of a GraphQL error entry.  See https://spec.graphql.org/June2018/#sec-Errors
+    /// <br/><br/>
+    /// Note that the field names must be encoded as camelCase
+    /// </summary>
+    public class ParsedError
+    {
+        // The following descriptions are copied from https://spec.graphql.org/June2018/#sec-Errors
+
+        /// <summary>
+        /// A description of the error intended for the developer as a guide to understand and correct the error
+        /// </summary>
+        public string Message { get; set; }
+
+        /// <summary>
+        /// A list of locations, where each location is a map with the keys 'line' and 'column', both positive numbers
+        /// starting from 1 which describe the beginning of an associated syntax element
+        /// </summary>
+        public IEnumerable<ErrorLocation> Locations { get; set; }
+
+        /// <summary>
+        /// A list of path segments starting at the root of the response and ending with the field associated with the error.
+        /// Path segments that represent fields should be strings, and path segments that represent list indices should be 0‐indexed integers.
+        /// If the error happens in an aliased field, the path to the error should use the aliased name, since it represents a path in the response, not in the query.
+        /// </summary>
+        public IEnumerable<object> Path { get; set; }
```

Comment:
Note that the current implementation of the `Path` property does not meet spec, as it returns strings for indices rather than numbers as shown in the spec description and reference sample code.  This code is prepped to rectify that error, but will require further changes.

Reply:
Let's merge #1657 first.

Reply:
Fine with me.  FYI, there are no conflicts between the two PRs (besides apiapprovals).  Either can be merged first with no changes to the other PR.